.github/actions/templates/getParameterFiles/action.yml

```diff
@@ -30,5 +30,10 @@ runs:
         # Get the list of parameter file paths
         $parameterFilePaths = Get-ModuleParameterFiles @functionInput -Verbose
         # Output values to be accessed by next jobs
-        Write-Output ('::set-output name=parameterFilePaths::{0}' -f ($parameterFilePaths | ConvertTo-Json -Compress))
+        $compressedOutput = $parameterFilePaths | ConvertTo-Json -Compress
+        if($compressedOutput -notmatch "\[.*\]") {
+          $compressedOutput = "[$compressedOutput]"
+        }
+        Write-Verbose "Publishing output: $compressedOutput" -Verbose
+        Write-Output "::set-output name=parameterFilePaths::$compressedOutput"
```

Comment:
As you teach me 
```suggestion
        Write-Output ('::set-output name=parameterFilePaths::{0}' -f $compressedOutput)
```

Reply:
If we have more than one parmaeter - or - we'd need to access a property of the parameter I'd definitely agree. In this case however I'd suggest to use the 'easy' syntax.